Author: Daniel F. Dickinson <cshored@thecshore.com>
Date:   Tue Jan 16 03:01:57 2018 -0500

    mountd: Fix mountd build error due to stddef.h redefintion with musl

Without this patch the build of mountd complains about redefintion of
various #defines from stddef.h (from different files) in musl.
We prevent this by removing #includes that are also included from
another file (and are hence included twice, including stddef.h).

Signed-of-by: Daniel F. Dickinson <cshored@thecshore.com>
Index: git/include/list.h
===================================================================
--- git.orig/include/list.h
+++ git/include/list.h
@@ -1,7 +1,6 @@
 #ifndef _LINUX_LIST_H
 #define _LINUX_LIST_H
 
-#include <stddef.h>
 /**
  * container_of - cast a member of a structure out to the containing structure
  * @ptr:	the pointer to the member.
Index: git/autofs.c
===================================================================
--- git.orig/autofs.c
+++ git/autofs.c
@@ -1,7 +1,6 @@
 #include <sys/types.h>
 #include <linux/types.h>
 #include <paths.h>
-#include <limits.h>
 #include <time.h>
 
 #include <stdio.h>
